specification.md

```diff
@@ -54,6 +57,11 @@ where:
     -   https://theupdateframework.com/Root/v1.0.5
     -   etc...
 
+*   KEYID is an optional, unauthenticated hint indicating what key was used to
+    sign the message. It **must not** be used for security decisions.
```

Comment:
```suggestion
    sign the message. It **MUST NOT** be used for security decisions.
```
In TUF we uppercase terms from [RFC 2119](https://tools.ietf.org/html/rfc2119) and refer readers there for interpretation guidelines.

Reply:
Thanks! Done.